src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -170,7 +174,7 @@ private CentralPackageVersion ToCentralPackageVersion(string packageId, string v
         public override async Task<IEnumerable<PackageReference>> GetInstalledPackagesAsync(CancellationToken token)
         {
             // Settings are not needed for this purpose, this only finds the installed packages
-            return GetPackageReferences(await GetPackageSpecAsync(NullSettings.Instance));
+            return await GetPackageReferencesAsync(await GetPackageSpecAsync(NullSettings.Instance));
```

Comment:
I'd split this into two lines. There's no performance benefit to avoiding creating a function local variable.